keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -312,30 +305,46 @@ proposed resize operation for a given resource.  Any time the
 `Pod.Status.ContainerStatuses[i].Resources` field, this new field explains why.
 
 This field can be set to one of the following values:
-* `Proposed` - the proposed resize (in Spec...Resources) has not been accepted or
-  rejected yet. Desired resources != Allocated resources.
-* `InProgress` - the proposed resize has been accepted and is being actuated. A new resize request
-  will reset the status to `Proposed`. Desired resources == Allocated resources != Actual resources.
+* `InProgress` - the proposed resize has been accepted and is being actuated. A `Deferred` or
+  `Infeasible` resize will take precedence over `InProgress`.
+  Desired resources == Allocated resources != Actual resources.
 * `Deferred` - the proposed resize is feasible in theory (it fits on this node)
   but is not possible right now; it will be re-evaluated.
   Desired resources != Allocated resources.
 * `Infeasible` - the proposed resize is not feasible and is rejected; it will not
   be re-evaluated. Desired resources != Allocated resources.
 * (no value) - there is no proposed resize.
   Desired resources == Allocated resources == Acutal resources.
+* `Error` - if an error occurs while actuating the resize (see [Memory Limit Decreases](#memory-limit-decreases)
+  for an example), then the resize status will be set to `Error` and an event will report the
+  details. The error state behaves similarly to `InProgress`, and the allocated resize will be
+  retried on the next pod sync.
 
 Any time the apiserver observes a proposed resize (a modification of a
 `Spec...Resources` field), it will automatically set this field to `Proposed`.
```

Comment:
Should we modify this sentence along with deprecating `Proposed`?

This line also mentions `Proposed`:
https://github.com/kubernetes/enhancements/blob/e49b717dcf72e2484ffead763a0cd06bbf7a5f5d/keps/sig-node/1287-in-place-update-pod-resources/README.md?plain=1#L261-L262

Reply:
Thanks, fixed.